ui/revenuecatui/src/main/kotlin/com/revenuecat/purchases/ui/revenuecatui/composables/PurchaseButton.kt

```diff
@@ -150,7 +150,7 @@ private fun PurchaseButton(
                         fontWeight = FontWeight.SemiBold,
                         allowLinks = false,
                         modifier = Modifier
-                            .padding(vertical = UIConstant.defaultVerticalSpacing / 3)
+                            .padding(vertical = UIConstant.defaultVerticalSpacing * 2 / 3)
```

Comment:
This doesn't exactly match back up to the original size but it's pretty close. I think it's difficult to match it exactly since the components within markdown before perform some custom painting that was altering the sizes slightly... Lmk what you think!